starboard/rdk/launcher.py

```diff
@@ -158,8 +158,8 @@ def _InitPexpectCommands(self):
 
     # test output tags
     self.test_complete_tag = 'test suites ran.'
-    self.test_failure_tag = 'tests, listed below'
-    self.test_success_tag = 'succeeded'
```

Comment:
Why not use the `test_success_tag` like in the raspi launcher this was borrowed from?

Reply:
The "succeeded" doesn't always come in this WPEFramework log mechanism.

Reply:
Are we missing some log lines? That seems concerning.

Reply:
I will recheck this part. Since I have this CL broken into 3. I will revisit this 10-line/ succeeded tag waiting in the other PR later.

Reply:
Will have a separate CL for this part.